test/e2e_node/jenkins/jenkins-ci-ubuntu.properties

```diff
@@ -6,6 +6,7 @@ GCE_ZONE=us-central1-f
 GCE_PROJECT=k8s-jkns-ubuntu-node
 CLEANUP=true
 GINKGO_FLAGS='--skip="\[Flaky\]|\[Serial\]"'
+TEST_ARGS='--feature-gates=KubeletConfigFile=true'
```

Comment:
What does it mean to enable this without passing a config file arg? Is there a reason to keep requiring a feature gate in addition to a flag for a soon to be beta feature?

Reply:
e2e node test framework sets up the file if the gate is on

Reply:
That's unexpected as well. We really don't want extra magic layered on by the test framework. For example, this means we can't test the "feature gate on but no file arg passed does nothing" case

Reply:
In this case, the feature gates are actually flags to the test runner, which forwards them to the binaries. Let's continue this conversation on #56995